presto-main/src/main/java/io/prestosql/testing/LocalQueryRunner.java

```diff
@@ -366,6 +366,7 @@ private LocalQueryRunner(Session defaultSession, FeaturesConfig featuresConfig,
 
```

Comment:
Instead of "Minor fix in LocalQueryRunner", make the description more explicit: "Add testing catalog to LocalQueryRunner"